src/tests/test_method.py

```diff
@@ -304,6 +304,10 @@ def testValueParamsArgs(self):
         self.assertTrue(result[1] == 2)
         self.assertTrue(result[2] == 3)        
 
+    def testNonParamsArrayInLastPlace(self):
```

Comment:
I just tried it, but C# won't let me have both

   TestNonParamsArrayInLastPlace(int i1, params int[] ints)

and

   TestNonParamsArrayInLastPlace(int i1, int[] ints)

It says they have the same name and parameter types, error CS0111.